src/bin/triplydb-run-pipeline.ts

```diff
@@ -0,0 +1,110 @@
+#!/usr/bin/env node
+
+import { program } from "commander";
+import colors from "colors";
+import App from "../App.js";
+import Pipeline, { createPipeline, defaultLogger } from "../Pipeline.js";
+import { Models } from "@triply/utils";
+import fs from "fs-extra";
+
+let defaultTriplyDBToken = process.env["TRIPLYDB_TOKEN"];
+let defaultTriplyDBAccount = process.env["TRIPLYDB_ACCOUNT"];
+let defaultHttpsProxy = process.env["HTTPS_PROXY"];
+let defaultHttpProxy = process.env["HTTP_PROXY"];
+const timeLog = (...args: string[]) => console.info(`[${new Date().toISOString()}]`, ...args);
+
+const command = program
+  .createCommand("run-pipeline")
+  .summary("Run a TriplyDB pipeline")
+  .usage("[options] <json config file>")
+  .description(
+    "Run a TriplyDB pipeline. TriplyDB pipelines currently supports executing speedy construct queries. A pipeline can run for an arbitrary duration, and for arbitrarey result sizes.",
+  )
+  .addHelpText(
+    "after",
+    `
+    Sample json config:
+    {
+      "queries": [{
+          "name": "accountName/queryName", (Required)
+          "priority": 1, (Optional)
+
+      },{
+          "name": "accountName/queryName",
+          "version": 2 (Optional)
+      }],
+      "sourceDataset": "accountName/datasetName", (Required)
+      "targetDataset": "accountName/datasetName", (Required)
+      "targetGraphName": "graph:default", (Optional)
+      "version": 0.1 (Required)
+    }
+    `,
+  )
+  .option("-t, --token <token>", "TriplyDB access token (default: $TRIPLYDB_TOKEN)")
+  .option(
+    "-a, --account <account>",
+    "Account where query job is created and stored. (default: the account that owns the token, or $TRIPLYDB_ACCOUNT)",
```

Comment:
Is it indeed the case that the owner of the token has precedence over the `TRIPLYDB_ACCOUNT` env variable?

Reply:
Actually guessing from the code, I think the precedence is the other way round. If that's indeed the case, I would say something like this (_at least_ change the order to match the code):

> 1st default: $TRIPLYDB_ACCOUNT; 2nd default: the account that owns the token